src/main/java/com/avispl/symphony/dal/logitech/collabos/LogitechCollabOsCommunicator.java

```diff
@@ -48,7 +48,7 @@
  *
  * @author Kevin / Symphony Dev Team<br>
  * Created on 1/3/2024
- * @since 1.0.0
+ * @since 1.0.1
```

Comment:
I don't think you need to update this tag here :) The class exists since v1.0.0, right?